### PR TITLE
Added: contribution guidelines

### PR DESCRIPTION
**Pull request by costomato**

### CODE REVIEW

**Comment by Suyious** [comment]
@costomato Under review.

**Comment by costomato** [comment]
> @costomato Under review.

are u fucked in your head???

File: README.md [hunk 1]
```diff
@@ -54,8 +54,22 @@ The backend listens for the following WebSocket events:
...
 
+
 ## Contributing
-Contributions are welcome! Feel free to open issues, fork the repository, and submit pull requests.
+Thank you for contributing to this open-source project! We strive to create a safe and collaborative environment at Purrr.chat. Please follow these contribution guidelines:
+
+### Contribution Rules
+1. **Fork the Repository**: Start by forking the repository before making any changes. 
+2. **Create a New Branch**: Use your name to create a new branch and start implementing your changes.
+3. **Push Your Changes**: Once your changes are ready, push them to your fork and wait for a review.
```
[the remark this answers is not in the document]
Comment (costomato) [reply README.md:64]: sounds good

**Review by Suyious — APPROVED** [review]
Looks Good. The efforts are appreciated.

**Pull request merged**